rs/nns/governance/src/governance.rs

```diff
@@ -5221,10 +5221,23 @@ impl Governance {
         let conversion_result = SnsInitPayload::try_from(ApiCreateServiceNervousSystem::from(
             create_service_nervous_system.clone(),
         ));
-        if let Err(err) = conversion_result {
+
+        #[allow(unused_variables)]
+        let validated = match conversion_result {
+            Ok(validated) => validated,
+            Err(err) => {
+                return Err(GovernanceError::new_with_message(
+                    ErrorType::InvalidProposal,
+                    format!("Invalid CreateServiceNervousSystem: {}", err),
+                ));
+            }
+        };
+
+        #[cfg(not(any(feature = "canbench-rs", feature = "test")))]
```

Comment:
Can we use our usual flag pattern? Having them all in one place is nice. Makes it easier to correctly+quickly change configuration. I know this is temporary, but most flags are also temporary too.

Reply:
changing this.